torchvision/prototype/features/_keypoint.py

```diff
@@ -0,0 +1,67 @@
+from typing import Dict, Any, Tuple, Optional, Sequence, Union, Collection
+
+import torch
+from torchvision.prototype.utils._internal import StrEnum
+
+from ._feature import Feature, DEFAULT
```

Comment:
What is `DEFAULT` here? From what I remember, this is currently missing from your latest proposal for the API at the [feature branch](https://github.com/pytorch/vision/tree/revamp-prototype-features-transforms). Coming into this work to review without much context, I"m very confused on what the end proposal is because we have simply too many open PRs with conflicting features. As mentioned above, we should dedicate time to clean up, close/merge PRs before opening new ones, prioritizing those that describe the skeleton of the API and deprioritizing those that add more features.